packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -58,7 +54,7 @@ export class StartImportFlow {
 	async clickButton( text: string ): Promise< void > {
 		const selector = selectors.button( text );
 
-		await this.page.waitForSelector( selector );
+		await this.page.locator( selector ).waitFor();
 		await this.page.click( selector );
```

Comment:
This is equivalent to:

```suggestion
		await this.page.locator( selector ).click();
```

`click()` internally waits until the element is clickable, so we don't need to call `waitFor()` explicitly.